### PR TITLE
Fix codes

### LINKED ISSUE #121: Some missing codes in inventoryids_iso_gcode.csv
1906,gny,
1970,qpt,
1999,adc,
2143,psx,
2150,yrm,

resolved to:

1906,pue,puel1244
1970,gvp,para1315
1999,mcd,shar1245
2143,tuo,pisa1245
2150,wca,yaro1235

- there is no ISO code for Pisamera, so I took the "language" which has it as a dialect -- here Glottolog is more detailed than ISO/Ethno
- yaro1235 is more detailed than the corresponding qca

### PR DESCRIPTION
**Pull request by bambooforest**
deals with and fixes #121

### CODE REVIEW

File: mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv [hunk 1]
```diff
@@ -2924,3 +2924,6 @@ InventoryID	LanguageCode	Glottocode	BibtexKey	Source
 2154	yup	yukp1241	2154_Meira2003	saphon
 2154	yup	yukp1241	2154_Oquendo2004	saphon
 2155	trn	trin1274	2155_RoseNULL	saphon
+2156	sje	pite1240	sje_wilbur2009_phon	uz
+2157	sme	nort2671	sme_sme_BallsOddenRich2005.pdf	uz
+2158	bxr	russ1264	bxr_sanzheev1962	ea
```
Comment (drammock) [inline mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv]: missing newline (see the little red icon after "ea"?)

File: mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv [hunk 2]
```diff
@@ -2069,7 +2069,7 @@ InventoryID	LanguageCode	Glottocode	BibtexKey	Source
 1395	afr	afri1274	1395_afr	gm
 1396	agj	argo1244	1396_hertzon	gm
 1397	har	hara1271	1397_hertzon	gm
-1398	din		1398_din	gm
+1398	din	dink1262	1398_din	gm
```
Comment (drammock) [inline mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv]: this one isn't in #121, is it from some other issue that we should also be closing with this PR?

**Comment by drammock** [comment]
Can you explain why we need the UPSID feature set in there?  Other than that and my inline comments, LGTM.

File: mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv (hunk 2 above)
Comment (bambooforest) [inline mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv]: a quick search didn't show anything. might have just been missed.

File: mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv (hunk 1 above)
Comment (bambooforest) [inline mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv]: my bad. i also removed those last few lines -- testing adding a new data source.

**Comment by bambooforest** [comment]
because i pushed them at some point to my master branch and they're getting dragged along. removed.

**Pull request merged**